static/partition-view.html

```diff
@@ -40,6 +40,10 @@
       <div class="logo">
         <img src="/icons/logo_ocsf.svg">
         <span>Learn OCaml</span>
+        <select id="Hide tokens">
+          <option value="nicknames">Show nicknames</option>
```

Comment:
It's true that the `<select>` dropdown will make it possible to add new options later on if need be.

But starting from this PR #540, I'd like you also add another option:

```suggestion
          <option value="anon">Hide PII<option>
          <option value="nicknames">Show nicknames</option>
```

that completely hides Personally Identifiable Information, and just output something like `[%i"Student"]` (no need to implement something more involved with consecutive numbers or so!)